lib/core/ui/dashboard/home/home_score_card.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:live_score/core/theme/app_theme.dart';
+import 'package:live_score/features/dashboard/cubit/dashboard_cubit.dart';
+
+class HomeScoreCard extends StatelessWidget {
+  const HomeScoreCard(
+      {super.key,
+      required this.homeTeam,
+      required this.awayTeam,
+      required this.homeScore,
+      required this.awayScore,
+      required this.status,
+      required this.date,
+      required this.homeLogo,
+      required this.awayLogo});
+  final String homeTeam;
+  final String awayTeam;
+  final int? homeScore;
+  final int? awayScore;
+  final String status;
+  final String date;
+  final String? homeLogo;
+  final String? awayLogo;
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<DashboardCubit, DashboardState>(
+      builder: (context, state) {
+        if (state is Success) {
```

Comment:
If you are checking the state here, you should create two separate widgets: one for success, other for loading. These widgets should go into separate files.